api/grpcserver/admin_service.go

```diff
@@ -128,3 +130,27 @@ func (a AdminService) EventsStream(req *pb.EventStreamRequest, stream pb.AdminSe
 		}
 	}
 }
+
+func (a AdminService) PeerInfo(context.Context, *empty.Empty) (*pb.PeerInfoResponse, error) {
```

Comment:
will repost so that comment is not lost

> on some nodes we may have up to 10_000 peers. i would like to avoid memory spikes when reading peers from them, if it would be the case then it is better to switch to streaming